### PR TITLE
Link to contributing and maintainer guides from README

### PR DESCRIPTION
**Pull request by filmaj**
The same as the python-slack-sdk PR I sent earlier, just trying to make documentation a bit more clear for potential people wanting to contribute, run tests locally, etc. I tried to be specific about which version of OpenJDK to install - version 11 worked for me, but running with version 8 or 16 did not (I would get test failures).

### Category (place an `x` in each of the `[ ]`)

* [ ] **bolt** (Bolt for Java)
* [ ] **bolt-{sub modules}** (Bolt for Java - optional modules)
* [ ] **slack-api-client** (Slack API Clients)
* [ ] **slack-api-model** (Slack API Data Models)
* [ ] **slack-api-*-kotlin-extension** (Kotlin Extensions for Slack API Clients)
* [ ] **slack-app-backend** (The primitive layer of Bolt for Java)

## Requirements

Please read the [Contributing guidelines](https://github.com/slackapi/java-slack-sdk/blob/main/.github/contributing.md) and [Code of Conduct](https://slackhq.github.io/code-of-conduct) before creating this issue or pull request. By submitting, you are agreeing to the those rules.

### CODE REVIEW

File: .github/maintainers_guide.md [hunk 1]
```diff
@@ -4,13 +4,13 @@ This document describes tools, tasks and workflow that one needs to be familiar
 
 ## Tools
 
-Maintaining this project requires installing [OpenJDK](https://openjdk.java.net/) in your development environment. Also, [Apache Maven](https://maven.apache.org/) needs to be installed to build this project. All of the remaining tools are downloaded as `dependencies`, which means you'll have them available once you run `mvn test-compile` in a working copy of this repository.
+Maintaining this project requires installing [OpenJDK](https://openjdk.java.net/) in your development environment (for best results on Mac, use OpenJDK v11 via `brew install openjdk@11`). Also, [Apache Maven](https://maven.apache.org/) needs to be installed to build this project. All of the remaining tools are downloaded as `dependencies`, which means you'll have them available once you run `mvn test-compile` in a working copy of this repository.
```
Comment (seratch) [inline .github/maintainers_guide.md:7]: nit
```suggestion
Maintaining this project requires installing [OpenJDK](https://openjdk.java.net/) in your development environment (for best results on macOS, use OpenJDK v11 via `brew install openjdk@11`). Also, [Apache Maven](https://maven.apache.org/) needs to be installed to build this project. All of the remaining tools are downloaded as `dependencies`, which means you'll have them available once you run `mvn test-compile` in a working copy of this repository.
```

File: README.md [hunk 2]
```diff
@@ -121,3 +121,10 @@ If you get stuck, we’re here to help. The following are the best ways to get a
 ## Important Notice for jSlack users
 
 The [jSlack](https://search.maven.org/artifact/com.github.seratch/jslack) project has been transferred to [@slackapi](http://github.com/slackapi). The jSlack maintenance releases for security issues or major bugfixes will be continued at https://github.com/seratch/jslack-maintenance-releases .
+
+### Contributing
+
+We welcome contributions from everyone! Please check out our
+[Contributor's Guide](.github/contributing.md) for how to contribute in a
+helpful and collaborative way.
```
Comment (seratch) [inline README.md:129]:
```suggestion
We welcome contributions from everyone! Please check out our [Contributor's Guide](.github/contributing.md) for how to contribute in a helpful and collaborative way.
```

**Review by seratch — CHANGES_REQUESTED** [review]
Thanks for improving this! This PR is almost perfect but can you check my minor suggestions?

**Comment by seratch** [comment]
For the build failures, https://github.com/slackapi/java-slack-sdk/issues/779 is still an issue on this project's GitHub Actions execution. As a tentative workaround for it, we sometimes have to rerun failed jobs to make them successful.

**Review by seratch — APPROVED** [review]

**Comment by seratch** [comment]
GitHub Actions builds are pretty unstable today but the changes in this PR already look great to me 👍

**Comment by filmaj** [comment]
Great, assuming it passes the CI, I will squash the commits down to one and merge 😄 

Thank you for reviewing, I very much appreciate it 🥰

**Pull request merged**